unicodetools/src/main/java/org/unicode/text/utility/Utility.java

```diff
@@ -1468,7 +1468,10 @@ public static String getMostRecentUnicodeDataFile(
                 }
                 Path path = Settings.UnicodeTools.getDataPath(base, element);
                 if (path != null) {
-                    result = path.resolve(parts[2] + fileType).toString();
+                    var filePath = path.resolve(parts[2] + fileType);
+                    if (filePath.toFile().exists()) {
+                        result = path.resolve(parts[2] + fileType).toString();
+                    }
```

Comment:
why not
```suggestion
                    var filePath = path.resolve(parts[2] + fileType);
                    if (filePath.toFile().exists()) {
                        result = filePath.toString();
                    }
```
?

Reply:
Because I was tired when I got to that point, I think.